README.md

```diff
@@ -57,6 +53,30 @@ EOF
 oc create -f my-registry-secret.yml
 ```
 
+#### Red Hat Catalog Imagestreams
+
+The pipelines must pull the parent index images through the internal OpenShift
+registry to take advantage of the built-in credentials for Red Hat's terms-based
+registry. There is a one time configuration for each desired distribution catalog.
+
+```bash
+# Must be run once before certifying against the certified catalog.
```

Comment:
Why only once, since it imports the latest information? What if, meanwhile, somebody will push a newer version of the image, and the information will change?